app/views/expedition_types/index.html.slim

```diff
@@ -3,10 +3,12 @@ table
   tr
     th Expedition type
     th Description
+    th Longevity (sec)
```

Comment:
Jeśli to jest czas trwania wyprawy to bardziej pasuje mi słowo `Duration`, ale zostawiam to Państwu